kahuna/public/js/components/gu-date-range/gu-date-range.js

```diff
@@ -105,14 +108,11 @@ guDateRange.directive('guDateRange', [function () {
                 pikaEnd.show();
             });
 
-            $scope.$watch('pikaEndValue', function (pikaEndValue) {
-                var date = pikaEndValue && new Date(pikaEndValue);
-                pikaStart.setMaxDate(date || new Date());
```

Comment:
> Do you think it's a problem to have this spill over into the search functionality?

It feels a little odd to (potentially) search for uploads in the future, but no real strong opinion from me here, especially as it doesn't break anything. I'd release and get feedback.